cms/static/js/views/utils/create_course_utils.js

```diff
@@ -1,57 +1,21 @@
 /**
  * Provides utilities for validating courses during creation, for both new courses and reruns.
  */
-define(["jquery", "underscore", "gettext", "js/views/utils/view_utils"],
-    function ($, _, gettext, ViewUtils) {
-        return function (selectors, classes) {
-            var validateTotalCourseItemsLength, setNewCourseFieldInErr, hasInvalidRequiredFields,
-                createCourse, validateFilledFields, configureHandlers;
+define(["jquery", "gettext", "js/views/utils/view_utils", "js/views/utils/create_utils_base"],
+    function ($, gettext, ViewUtils, CreateUtilsFactory) {
+        "use strict";
+        var CreateCourseUtilsFactory = function (selectors, classes) {
 
-            var validateRequiredField = ViewUtils.validateRequiredField;
-            var validateURLItemEncoding = ViewUtils.validateURLItemEncoding;
+            // Needed for prototype functions.
+            this.selectors = selectors;
+            this.classes = classes;
 
```

Comment:
@cahrens Should be good to go now. If there are no other notes, I'll ping you once the build is green.